source/installation-guide/installing-wazuh-agent/hpux/wazuh_agent_sources_hpux.rst

```diff
@@ -0,0 +1,77 @@
+.. Copyright (C) 2019 Wazuh, Inc.
+
+.. _wazuh_agent_sources_hpux:
+
+HP-UX from sources
+==================
+
+This guide describes how to install the Wazuh agent from source code for HP-UX. For other operating systems, please check the list: :ref:`Install Wazuh agent <installation_agents>`.
+
+Installing Wazuh agent
+----------------------
+
+.. note:: All the commands described below need to be executed with root user privileges. Since Wazuh 3.5 it is necessary to have internet connection when following this process.
+
+1. Install development tools and compilers.
+
+   1.1 Install the package manager.
+
+     .. code-block:: console
+
+        # swinstall -s depothelper-2.10-hppa_32-11.31.depot \*
```

Comment:
Where can I found the file `depothelper`? Please indicate a step to download it before to use it.

Reply:
Done in 3e492705402a5b51a80ac21fa7240c8f73e35d01